project/src/types/offers-types.ts

```diff
@@ -0,0 +1,9 @@
+export type OffersTypes = {
```

Comment:
Тип надо описать полностью согласно структуре сервера
```suggestion
export type Offer = {
```